### ISSUE TITLE
Problem with central european summer time

### ISSUE THREAD

**Issue by NiklasLazinbee** [jelhan/croodle#569]
I have some problems with different time indications for the central european summertime (screenshot).
[Croodle_Timezone.pdf](https://github.com/jelhan/croodle/files/11225655/Croodle_Timezone.pdf)

**Comment by jelhan** [comment]
Thanks a lot for reporting.

Do you have a Croodle demonstrating the bug, which you could share publicly? Additionally the time zone used in your browser would be helpful.

**Comment by NiklasLazinbee** [comment]
Thanks for the reply. 

I have temporarily set up a Croodle here

http://croodle.salon-olga.de/

The timezone is Berlin, Germany

**Comment by jelhan** [comment]
Wasn't aware that this issue affects every Croodle created with central europan summer time. I was able to reproduce with this instance as well: https://www.systemli.org/poll/#/poll/xh8aZCuAcH/evaluation?encryptionKey=VB7ITF79bXMXSU4UNzMfk27bNaVxQQeOKAumSltM I will have a look. Thanks for reporting.

You can tear down http://croodle.salon-olga.de/ if only setup for this purpose. I'm sorry for the additional work. Should have tried to reproduce with another instance myself first.

**Comment by lefouk** [comment]
Same issue here: croodle.dnr.de/
Any solution in sight?
Thanks m.

**Comment by pgerstbach** [comment]
I have the same issue.

I did the following:
My browser/Mac is set to CEST. I created a new poll with time 12:00
On the same browser it is immediately showas at 11:00
I tested in both on Safari and Firefox.

When I open the same poll in Windows/Edge it prompts me that the poll was created in another timezone and in which it should be displayed. I didn't see this prompt on my Mac. If I open it in "my timezone" the dates are shown as 12:00 if I open in "original timezone" it is shown at 11:00.
